### DISCUSSION TITLE
Windows10 正常使用，iphone无法使用

### DISCUSSION THREAD

**Discussion by irvinefish** [getlantern/lantern-forum-cn#345] [Q&A]
windows 10能正常使用lantern,iPhone 不知从哪天就无法使用了，找不到填写账户的地方

**Comment by MaxMKMax** [comment]
@irvinefish 您好. 请问是空白无法填写吗? 还是完全找不到登入? 您可以使用iPhone帮我提交应用内报告吗? 我需要log来给科技部修复问题. 您提交之后请把 8 位数票号给我. 在邮件里有票号. 谢谢!

**Comment by irvinefish** [comment]
你好，应用内报告无法提交，填写后就找不到提交按钮 票号找到了：1878-6710
Mike K ***@***.***> 于2024年4月1日周一 16:41写道：
